src/data_rate.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package helium;
+
+enum data_rate {
+  SF12BW125 = 0;
+  SF11BW125 = 1;
```

Comment:
Google recommends UNSPECIFIED for the 0 default.  I'd recommend the same.
It can happen that the DataRate is unknown or unspecified.  SF12BW125 is the standard default for most plans so that might work for most plans, but SF12BW125 is an illegal DataRate for US915.

https://developers.google.com/protocol-buffers/docs/proto3#enum